_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: default
+title: OTel trace group
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OTel trace group
+
+The `OTel trace group` processor completes missing trace group related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend. The `OTel trace group` processor iedntifies the missing trace group information for a `spanId` by looking up the relevant fields in its root `span` stored in OpenSearch, or in the Amazon OpenSearch Service backend that the local Data Prepper host ingests.
```

Comment:
@dlvenable @oeyh Do we have a link to documentation that's not just the .java file on github?

Reply:
I don't find one. "span" is a common term in trace analytics though. If you don't want the link directly to code, you can probably replace it with "a model defined in Data Prepper to represent spans in traces"

By the way, there's a typo in "iedntifies".